Automattic-Tracks-iOS/ABTesting/ABTesting.swift

```diff
@@ -4,7 +4,6 @@ public enum Variation: Equatable {
     case control
     case treatment
     case other(String)
```

Comment:
It might be a good idea to do what @renanferrari did on https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1845 -- and make Treatment take a string since `other()` is just a different Treatment (for when we support multi-variate experiments).